### PR TITLE
doc: add more info in test.md

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

**1.Describe what this PR did**
Add more info in test.md.

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: docs/test/test.md [hunk 1]
```diff
@@ -10,28 +10,52 @@ This doc will give an introduction about the following three parts:
 
 # Organization of test
 
-Test in pouch could be divided into two parts [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description) and [integration testing](https://en.wikipedia.org/wiki/Integration_testing). More info could be get from the wiki page. For pouch developer, if your code is only used in a single module, then the unit test is enough. While if your code is called by multiple modules, integration tests are required. In pouch, both of them are developed using go language. More details could be got in [Unit Testing](#unit-testing) and [Integration Testing](#integration-testing).
+Test in pouch could be divided into following parts:
+
+* [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description)
+* [integration testing](https://en.wikipedia.org/wiki/Integration_testing)
+
+More info could be get from the wiki page.
```
Comment (allencloud) [inline docs/test/test.md]: s/get/gotten ?

File: docs/test/test.md [hunk 2]
```diff
@@ -10,28 +10,52 @@ This doc will give an introduction about the following three parts:
...
 
-Test in pouch could be divided into two parts [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description) and [integration testing](https://en.wikipedia.org/wiki/Integration_testing). More info could be get from the wiki page. For pouch developer, if your code is only used in a single module, then the unit test is enough. While if your code is called by multiple modules, integration tests are required. In pouch, both of them are developed using go language. More details could be got in [Unit Testing](#unit-testing) and [Integration Testing](#integration-testing).
+Test in pouch could be divided into following parts:
+
+* [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description)
+* [integration testing](https://en.wikipedia.org/wiki/Integration_testing)
+
+More info could be get from the wiki page. 
+
+For pouch developer, if your code is only used in a single module, then the unit test is enough. While if your code is called by multiple modules, integration tests are required. In pouch, both of them are developed using go language. More details could be got in [Unit Testing](#unit-testing) and [Integration Testing](#integration-testing).
```
Comment (allencloud) [inline docs/test/test.md]: s/got/gotten ?

File: docs/test/test.md [hunk 3]
```diff
@@ -10,28 +10,52 @@ This doc will give an introduction about the following three parts:
...
+Tests could be run through the target provided by `Makefile` in the root directory of pouch code. Also could be run manually.	
+To run the whole test, you could just run:
+
+```
+# make test
+```
+Please note that, in order to run test, the following prerequisites are needed:
+	
+	* golang is installed
+	* docker is installed
```
Comment (allencloud) [inline docs/test/test.md]: In the future, we will replace docker with pouch itself.

Comment (Letty5411) [reply docs/test/test.md]: Yes, when pouch support build option~

**Comment by allencloud** [comment]
LGTM

**Pull request merged**